.github/workflows/pr-check.yaml

```diff
@@ -300,10 +296,12 @@ jobs:
         run: yarn --frozen-lockfile
 
       - name: Run E2E smoke tests
+        env:
```

Comment:
I would not place this env. to the PR check yet. Please remove it. we will try with e2e-test-main first.